modules/alpine/main.go

```diff
@@ -120,15 +120,7 @@ func (m *Alpine) Container(ctx context.Context) (*dagger.Container, error) {
 		repos = wolfiRepositories()
 
 		basePkgs = []string{
-			// HACK: Wolfi will occasionally publish APKINDEX files that are missing required
-			// versions of glibc and related packages. This is a problem when one of them, like
-			// ld-linux, specifies a dependency on an exact version that is not present.
-			// This resolves itself usually within an hour, but to avoid fundamental breakage of
-			// builds we are currently pinning these packages for now.
-			"busybox=1.37.0-r49",
-			"glibc=2.42-r0",
-			"ld-linux=2.42-r0",
-			"libcrypt1=2.42-r0",
+			"busybox",
```

Comment:
This is technically tangential, but I needed it at various times to install some packages successfully. The workaround was added prior to when we bumped our go-apk dependency which fixed a ton of bugs. Won't be sure until we try but I'm hoping we may now longer need this workaround 🤞